src/components/AccordionComponent/ClassTrackerItems.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+//Sample Test Component to be replaced with Admin - class 
+// Streamline Types, Integrate the types later to a single file
+
+type ClassTrackerItemsType = {
+  title: string;
+  content: ReactNode;
+};
+
```

Comment:
we could create a model.ts to avoid writing this multiple times.
```Typescript
type ClassTrackerItemsType = {
  title: string;
  content: ReactNode;
};
```

Reply:
Yes, that's why I added a comment there "Streamline Types, Integrate the types later to a single file"  
I read an article that provides various ways to organize the types,  one way of organizing all types and interfaces is by storing them in one unique types.ts for the project file, another way is to have the types stored inside each component. Each approach has their pros and cons => Here is the link to the article https://www.becomebetterprogrammer.com/typescript-organizing-and-storing-types-and-interfaces/
It's something we can discuss in our calls :)

Reply:
I added `types.ts` in my PR that just got merged, maybe you can add it there? We may be able to clean it up a bit though later